personio-personnel-data-api-oa3.yaml

```diff
@@ -3983,3 +4105,700 @@ components:
+        metadata:
+          $ref: "#/components/schemas/PublicReportMetaData"
+        offset:
+          type: integer
+        limit:
+          type: integer
+        data:
+          $ref: "#/components/schemas/PublicColumnData"
+
+    PublicReportMetaData:
+      type: object
+      properties:
+        total_elements:
+          type: integer
+        current_page:
+          type: integer
+        total_pages:
+          type: integer
+
+    PublicReportData:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "Report"
+        attributes:
+          $ref: "#/components/schemas/PublicReportAttributes"
+
+    PublicReportItemsData:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "Report"
+        attributes:
+          $ref: "#/components/schemas/PublicReportItemsAttributes"
+
+    PublicColumnData:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "Column"
+        attributes:
+          type: array
+          items:
+            $ref: "#/components/schemas/PublicColumnAttributes"
+
+    PublicColumnAttributes:
+      type: object
+      properties:
+        attribute_id:
+          type: string
+          example: first_name
+        human_readable:
+          type: string
+          example: First Name
+        data_type:
+          type: string
+          example: "TEXT"
+
+    PublicReportAttributes:
+      type: object
+      properties:
+        id:
+          $ref: "#/components/schemas/UniqueIdentifier"
+        name:
+          type: string
+          example: Employee vacations past month
+        description:
+          type: string
+          example: Shows vacations of all employees that were taken past month
+        author_first_name:
+          type: string
+          description: Report author
+          example: Robert
+        author_last_name:
+          type: string
+          description: Report author
+          example: Sirano
+        type:
+          type: string
+          example: point_in_time
+          enum:
+            - point_in_time
+            - historical_data
+            - timeframe
+        employee_status:
+          type: string
+          example: currently_active
+          enum:
+            - currently_active
+            - active_at_given_date
+            - all
+        status:
+          type: string
+          example: up_to_date
+          enum:
+            - up_to_date
+            - updating
+            - update_failed
+        start_date:
+          type: string
+          format: date-time
+          example: "2019-04-01"
+        end_date:
+          type: string
+          format: date-time
+          example: "2019-04-10"
+        created_at:
+          type: string
+          format: date
+          example: "2019-04-10"
+        updated_at:
+          type: string
+          format: date
+          example: "2019-04-10"
```

Comment:
For all of these dates, doesn't it make sense to choose a format which contains the Timezone?

Reply:
None of these dates have a timezone. We use UTC by default, which is not stored in the DB column.

Reply:
I see - as long as the documentation reflect that, that is just as fine :)

Reply:
I have changed it. Thanks! 🚀